client/public/styles/home.scss

```diff
@@ -134,6 +178,10 @@
   height: 200px;
```

Comment:
em's are great! We eventually want to move away from using px entirely, if you'd like to do this in a later pr thats fine though.